packages/okta-vue/README.md

```diff
@@ -0,0 +1,223 @@
+  - [OAuth 2.0 Implicit Flow](https://tools.ietf.org/html/rfc6749#section-1.3.2)
+
+## Getting Started
+* If you do not already have a **Developer Edition Account**, you can create one at [https://developer.okta.com/signup/](https://developer.okta.com/signup/).
+* If you don't have a Vue app, or are new to Vue, please start with the [Vue CLI](https://github.com/vuejs/vue-cli) guide. It will walk you through the creation of a Vue app, creating [routers](https://router.vuejs.org/en/essentials/getting-started.html), and other application development essentials.
+
+### Add an OpenID Connect Client in Okta
+In Okta, applications are OpenID Connect clients that can use Okta Authorization servers to authenticate users.  Your Okta Org already has a default authorization server, so you just need to create an OIDC client that will use it.
+* Log into the Okta Developer Dashboard, click **Applications** > **Add Application**.
+* Choose **Single Page App (SPA)** as the platform, then submit the form the default values, which should look like this:
+
+| Setting             | Value                                          |
+| ------------------- | ---------------------------------------------- |
+| App Name            | My SPA App                                     |
+| Base URIs           | http://localhost:{port}                        |
+| Login redirect URIs | http://localhost:{port}/implicit/callback      |
+| Grant Types Allowed | Implicit                                       |
+
+After you have created the application there are two more values you will need to gather:
+
+| Setting       | Where to Find                                                                  |
+| ------------- | ------------------------------------------------------------------------------ |
+| Client ID     | In the applications list, or on the "General" tab of a specific application.   |
+| Org URL       | On the home screen of the developer dashboard, in the upper right.             |
+
+
+These values will be used in your Vue application to setup the OpenID Connect flow with Okta.
+
+## Installation
+
+This library is available through [npm](https://www.npmjs.com/package/@okta/okta-vue). To install it, simply add it to your project:
+
+```bash
+npm install --save @okta/okta-vue
+```
+
+### Configuration
+You will need the values from the OIDC client that you created in the previous step to instantiate the middleware. You will also need to know your Okta Org URL, which you can see on the home page of the Okta Developer console.
+
+In your application's [vue-router](https://router.vuejs.org/en/essentials/getting-started.html) configuration, import the `@okta/okta-vue` plugin and pass it your OpenID Connect client information:
+
+```typescript
+// router/index.js
+
+import Auth from '@okta/okta-vue'
+
+Vue.use(Auth, {
+  issuer: 'https://{yourOktaDomain}.com/oauth2/default',
+  client_id: '{client_id}',
+  redirect_uri: 'https://localhost:{port}/implicit/callback',
```

Comment:
For the redirect_uri, we are using `https` in this guide, but when you go through the setup in the okta dashboard, it defaults to `http`.  We should set it up to be the same, either way

Reply:
Good catch!